src/screens/GrantsScreen/styles.tsx

```diff
@@ -1,5 +1,20 @@
 import { StyleSheet } from "react-native";
 
-export default StyleSheet.create({ 
-
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#FAFAFA",
+    alignContent: "space-around",
+  },
+  grantsText: {
+    marginBottom: 30,
+    marginLeft: 24,
+    marginTop: 20,
+  },
+  innerContainer: {
+    flex: 1,
+    backgroundColor: "#FAFAFA",
+    alignItems: "center",
+    justifyContent: "space-between",
+  },
 });
```

Comment:
Make sure to delete src/types/mongoSchema.ts, it seems not important for this pr!